packages/blade/package.json

```diff
@@ -107,7 +107,10 @@
   },
   "dependencies": {
     "@babel/runtime": "7.16.5",
-    "use-presence": "1.1.0"
+    "use-presence": "1.1.0",
+    "@use-gesture/react": "10.2.24",
+    "body-scroll-lock": "4.0.0-beta.0",
```

Comment:
This version works with IOS https://github.com/willmcpo/body-scroll-lock/releases/tag/v4.0.0-beta.0

It's also been used by react-spring-bottomsheet should be fine since we are pinning it.